docs/install/virtual_machine.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: "Install A Zeppelin ready Virtual Machine"
-description: ""
+title: "Apache Zeppelin on Vagrant Virtual Machine"
+description: "Apache Zeppelin distribution provides a script directory for vagrant virtual machine. This script creates a virtual machine that launches a repeatable, known set of core dependencies required for developing Zeppelin."
```

Comment:
Apache Zeppelin provides a script for running a virtual machine for development though Vagrant. Script will create a virtual machine with core dependencies pre-installed, required for developing Apache Zeppelin.